test/soca/CMakeLists.txt

```diff
@@ -12,3 +12,108 @@ ecbuild_add_test(
         OBS_DIR=${OBS_DIR}
         PYTHONPATH=${PROJECT_BINARY_DIR}/ush:$ENV{PYTHONPATH}
 )
+
+# Prepare backgrounds
+# -------------------
+# 1 - create fake 3 hourly restarts under RESTART
+# 2 - store them with R2D2
+set( TESTDATA ${PROJECT_BINARY_DIR}/test/testdata )
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bkg)
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bkg/RESTART)
+ecbuild_add_test(
```

Comment:
I don't think we should be using ecbuild, see https://github.com/NOAA-EMC/GDASApp/blob/develop/test/atm/CMakeLists.txt on how to use `add_test` and `set_tests_properties`